<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnnotationWidget</class>
 <widget class="QWidget" name="AnnotationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>414</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="annotationTabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="pointsTab">
      <attribute name="title">
       <string>Points</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTableView" name="pointsTableView"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="spheresTab">
      <attribute name="title">
       <string>Spheres</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTableView" name="spheresTableView"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="linesTab">
      <attribute name="title">
       <string>Lines</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QTableView" name="linesTableView"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="anglesTab">
      <attribute name="title">
       <string>Angles</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QTableWidget" name="tableWidget_4">
         <column>
          <property name="text">
           <string>ID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Angle (rad)</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Angle (deg)</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Export ...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="clearAllButton">
       <property name="text">
        <string>Clear All</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>AnnotationWidget</receiver>
   <slot>exportCSV()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>27</x>
     <y>274</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearAllButton</sender>
   <signal>clicked()</signal>
   <receiver>AnnotationWidget</receiver>
   <slot>clearAllAnnotations()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>354</x>
     <y>276</y>
    </hint>
    <hint type="destinationlabel">
     <x>364</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>exportCSV()</slot>
  <slot>clearAllAnnotations()</slot>
 </slots>
</ui>
